onnxruntime/test/onnx/main.cc

```diff
@@ -335,35 +335,22 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
   // clang-format on
 
 #ifdef _WIN32
-  broken_tests["resnet50"] = "failed: type mismatch";
-  broken_tests["resnet50v2"] = "failed: type mismatch";
-  broken_tests["resnet101v2"] = "failed: type mismatch";
-  broken_tests["resnet101v2"] = "failed: type mismatch";
-  broken_tests["resnet152v2"] = "failed: type mismatch";
   broken_tests["tf_inception_resnet_v2"] = "failed: type mismatch";
-  broken_tests["tf_inception_v3"] = "failed: type mismatch";
   broken_tests["tf_inception_v4"] = "failed: type mismatch";
-  broken_tests["tf_resnet_v1_50"] = "failed: type mismatch";
-  broken_tests["tf_resnet_v2_50"] = "failed: type mismatch";
   broken_tests["tf_resnet_v1_101"] = "failed: type mismatch";
   broken_tests["tf_resnet_v1_152"] = "failed: type mismatch";
   broken_tests["tf_resnet_v2_101"] = "failed: type mismatch";
   broken_tests["tf_resnet_v2_152"] = "failed: type mismatch";
-
   broken_tests["vgg19"] = "failed: bad allocation";
   broken_tests["tf_nasnet_large"] = "failed: bad allocation";
   broken_tests["tf_pnasnet_large"] = "failed: bad allocation";
-
+  broken_tests["zfnet512"] = "failed: bad allocation";
```

Comment:
Is this a new failure? Why did it pass before?

Reply:
Does this only fail on x86? Didn't we fix that issue before? This used to be named vgg16 right?

Reply:
Yes I found new failed cases - they fail the test 7 out of 10 times, not always.